web/src/main.js

```diff
@@ -260,11 +263,22 @@ d3.select('#checkbox-colorblind').on('change', () => {
   dispatchApplication('colorBlindModeEnabled', !getState().application.colorBlindModeEnabled);
 });
 
+// update Theme
+let theme = themes.bright;
+function updateTheme() {
+  if (getState().application.brightModeEnabled) {
+    theme = themes.bright;
+  } else {
+    theme = themes.dark;
+  }
+  if (typeof zoneMap !== 'undefined') zoneMap.setTheme(theme);
+}
 
 // Start initialising map
 try {
-  zoneMap = new ZoneMap('zones', { zoom: 1.5 })
+  zoneMap = new ZoneMap('zones', { zoom: 1.5, theme })
     .setCo2color(co2color)
+    .setTheme(theme)
```

Comment:
seems unnecessary. I think you can remove as you already initialise the map with the theme